android/gradlew

```diff
@@ -1,7 +1,7 @@
-#!/usr/bin/env sh
```

Comment:
I should've asked this earlier: does gradlew get auto-generated?
I wonder why the shebang changed sh's path. The latter is a bit less robust AFAIK (some systems don't have sh under `/bin`)

Reply:
It does get automatically generated when creating a new plugin, which is what I've done for migrating to Capacitor 4. I'm not sure why this has changed but it hasn't caused any issues for me personally while building or testing the updated plugins in-app

Reply:
Ah ok I see. And yeah, most (POSIX) systems would have /bin/sh but it's encouraged best practice to use /usr/bin/env